src/main/scala/outwatch/dom/DomTypes.scala

```diff
@@ -65,26 +56,34 @@ trait Tags
   with FormTags[GenericVNode, VNode]
   with SectionTags[GenericVNode, VNode]
   with TableTags[GenericVNode, VNode]
-  with TagsCompat
   with VNodeBuilder
   with TagHelpers
+
+@deprecated("Use dsl.tags instead", "0.11.0")
 object Tags extends Tags
+                    with TagsCompat
 
 trait TagsExtra
   extends DocumentTags[GenericVNode, VNode]
   with MiscTags[GenericVNode, VNode]
   with VNodeBuilder
-object TagsExtra extends TagsExtra
 
 trait Attributes
   extends Attrs
   with ReflectedAttrs
   with Props
   with Events
-  with Styles
-  with AttributesCompat
+  with AttributesExtra
+  with AttributeHelpers
```

Comment:
minor nitpick: can we merge `AttributesExtra` and `AttributeHelpers`

Reply:
Yes, good idea!